test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2518,6 +2518,131 @@ def forward(self, x, y):
+        z = torch.tensor([1], dtype=torch.uint8)
+        self.run_test(AddModule(), (x, y))
+        self.run_test(SubModule(), (x, y))
+        self.run_test(MulModule(), (x, y))
+        self.run_test(DivModule(), (x, y))
+        self.run_test(PowModule(), (x, z))
+
+        x = torch.tensor([2, 3, 5], dtype=torch.int8)
+        y = torch.tensor([2, 3, 5], dtype=torch.int8)
+        z = torch.tensor([1], dtype=torch.int8)
+        self.run_test(AddModule(), (x, y))
+        self.run_test(SubModule(), (x, y))
+        self.run_test(MulModule(), (x, y))
+        self.run_test(DivModule(), (x, y))
+        self.run_test(PowModule(), (x, z))
+
+        x = torch.tensor([2, 3, 5], dtype=torch.int16)
+        y = torch.tensor([2, 3, 5], dtype=torch.int16)
+        z = torch.tensor([1], dtype=torch.int16)
+        self.run_test(AddModule(), (x, y))
+        self.run_test(SubModule(), (x, y))
+        self.run_test(MulModule(), (x, y))
+        self.run_test(DivModule(), (x, y))
+        self.run_test(PowModule(), (x, z))
+
+        x = torch.tensor([2, 3, 5], dtype=torch.uint8)
+        y = torch.tensor([2, 3, 5], dtype=torch.float32)
+        z = torch.tensor([1], dtype=torch.float64)
+        self.run_test(AddModule(), (x, y))
+        self.run_test(SubModule(), (x, y))
+        self.run_test(MulModule(), (x, y))
+        self.run_test(DivModule(), (x, y))
+        self.run_test(PowModule(), (x, z))
+
+        x = torch.tensor([2, 3, 5], dtype=torch.uint8)
+        y = torch.tensor([2, 3, 5], dtype=torch.int64)
+        z = torch.tensor([1], dtype=torch.int32)
+        self.run_test(AddModule(), (x, y))
+        self.run_test(SubModule(), (x, y))
+        self.run_test(MulModule(), (x, y))
+        self.run_test(DivModule(), (x, y))
+        self.run_test(PowModule(), (x, z))
+
+    # fmod was added in version 10
+    @skipIfUnsupportedMinOpsetVersion(10)
+    @skipIfUnsupportedMaxOpsetVersion(13)
+    def test_mod_with_low_precision(self):
+        class ModModule(torch.nn.Module):
+            def forward(self, x, y):
+                return torch.fmod(x, y).to(dtype=torch.long)
+
+        x = torch.tensor([2, 3, 5], dtype=torch.uint8)
+        y = torch.tensor([2, 3, 5], dtype=torch.uint8)
+        self.run_test(ModModule(), (x, y))
+
+        x = torch.tensor([2, 3, 5], dtype=torch.int8)
+        y = torch.tensor([2, 3, 5], dtype=torch.int8)
+        self.run_test(ModModule(), (x, y))
+
+        x = torch.tensor([2, 3, 5], dtype=torch.int16)
+        y = torch.tensor([2, 3, 5], dtype=torch.int16)
+        self.run_test(ModModule(), (x, y))
+
+        x = torch.tensor([2, 3, 5], dtype=torch.uint8)
+        y = torch.tensor([2, 3, 5], dtype=torch.int32)
+        self.run_test(ModModule(), (x, y))
+
+        x = torch.tensor([2, 3, 5], dtype=torch.uint8)
+        y = torch.tensor([2, 3, 5], dtype=torch.float64)
+        self.run_test(ModModule(), (x, y))
+
+    @unittest.skip("Gemm operator only support float/double in ONNX")
```

Comment:
What is the value of having these tests that are always skipped?
If we expect to be able to enable it at some point, could you please add a link to a tracking issue for the work needed to enable it?
If we don't expect to be able to enable it, can we just remove it?

Reply:
Since Gemm is also one of standardOps, this test added mainly for reminding. It's very hardly when gemm op encounter such a problem. I think we can remove it.

Reply:
Do you mean "handy" instead of "hardly"?
What if we instead add checks about the input types into the Gemm symbolic function, and add a unit test for those checks?

Reply:
I mean it may be rarely to meet this situation. Cause we usually call **Gemm** in symbolic function mm() and addmm().

However, servaral functions related in PyTorch [torch.mm](https://pytorch.org/docs/master/generated/torch.mm.html?highlight=mm#torch.mm), [torch.andmm](https://pytorch.org/docs/master/generated/torch.addmm.html#torch-addmm),  
 [torch.matmul ](https://pytorch.org/docs/master/generated/torch.matmul.html?highlight=matmul#torch.matmul) both are only for inputs of type **FloatTensor** or **DoubleTensor**. This is designed to achieve better performance [TensorFloat-32(TF32) on Ampere devices](https://pytorch.org/docs/master/notes/cuda.html#tensorfloat-32-tf32-on-ampere-devices). Maybe it could remove those lowprecision type test?